shared/src/popup.css

```diff
@@ -18,7 +19,7 @@ body {
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  width: 483px;
+  width: 500px;
```

Comment:
These changes make the popup look consistent throughout mobile, desktop, and Firefox + Chrome.